cordova-lib/src/plugman/fetch.js

```diff
@@ -135,13 +135,32 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
             var splitVersion = plugin_src.split('@');
             var newID = pluginMapperotn[splitVersion[0]];
             if(newID) {
-                events.emit('warn', 'Notice: ' + splitVersion[0] + ' has been automatically converted to ' + newID + ' to be fetched from npm. This is due to our old plugins registry shutting down.');
                 plugin_src = newID;
                 if (splitVersion[1]) {
                     plugin_src += '@'+splitVersion[1];
                 }
-            } 
-            return registry.fetch([plugin_src])
+            }
+            var P, skipCopyingPlugin;
+            plugin_dir = path.join(plugins_dir, splitVersion[0]);
```

Comment:
> In fact, cordova-plugin.js invokes plugman-fetch first and then plugman-install.

This is probably excess and could be reworked to just call `plugman/install` as it calls `tryFetch` anyway.

> because plugman-install can be invoked directly from plugin CLI

I guess you meant `plugman/fetch`, right? This might be a problem, but since this is user command, could we shift the responsibility for this to the user?